extensions/default/src/DicomWebDataSource/wado/retrieveMetadataLoaderAsync.js

```diff
@@ -74,7 +74,7 @@ export default class RetrieveMetadataLoaderAsync extends RetrieveMetadataLoader
     return sortStudySeries(
       naturalized,
       sortCriteria ||
-        sortingCriteria.seriesSortCriteria.seriesInfoSortingCriteria,
+      sortingCriteria.seriesSortCriteria.seriesInfoSortingCriteria,
```

Comment:
The sorting criteria should be globally configured, although a better choice might be to get it from the customization framework so that it can be set in the mode itself.